app/api/admin_translations.py

```diff
@@ -0,0 +1,23 @@
+from flask import send_file, Blueprint
+import shutil
+import uuid
+import tempfile
+import os
+
+admin_blueprint = Blueprint('admin_blueprint', __name__, url_prefix='/admin/content/translations/all')
+temp_dir = tempfile.gettempdir()
+translations_dir = 'app/translations'
+
+@admin_blueprint.route('/', methods=['GET'])
+def download_translations():
+    """Admin Translations Downloads"""
+    uuid_literal = uuid.uuid4()
+    zip_file = "translations{}".format(uuid_literal)
+    shutil.make_archive(zip_file, "zip", translations_dir)
+    shutil.move(zip_file+'.zip', temp_dir)
```

Comment:
Extract `zip_file+'.zip'` and replace occurances

Reply:
Taken care of.